controller/templates/controller/index.html

```diff
@@ -45,4 +45,8 @@ <h5>Finite State Machine</h5>
       {% include "main/messages.html" with topic="ERS" %}
     </div>
   </div>
+  <div id="arguments-dialog"
+       _="on htmx:afterSettle call (first of me).showModal()">
```

Comment:
Might this be slightly clearer if it was also done with an `on load` trigger as part of the htmx response?

Reply:
I'm not sure what you mean with this or how to address it. Could you elaborate?

Reply:
I've addressed all your comments but for this one.

Reply:
The idea was that rather than having an element in the main page that monitors htmx requests then opens the modal, `arguments_dialog.html` could contain an element with an `on load` hyperscript snippet to open the modal when it is first added to the page.

Reply:
Got it. Let's see what I can do.

Reply:
OK, have a look at [70d4f9e](https://github.com/ImperialCollegeLondon/drunc_ui/pull/241/commits/70d4f9e7dd9d59b8ce3e94b0ddda645538642912) and let me know what you think. Now is way more self contained - and it works.